back/engines/commercial/bulk_import_ideas/app/services/bulk_import_ideas/exporters/idea_html_pdf_form_exporter.rb

```diff
@@ -0,0 +1,96 @@
+# frozen_string_literal: true
+
+module BulkImportIdeas::Exporters
+  class IdeaHtmlPdfFormExporter < IdeaHtmlFormExporter
+    def format
+      'pdf'
+    end
+
+    def mime_type
+      'application/pdf'
+    end
+
+    def filename
+      'form.pdf'
+    end
+
+    # Render the form from HTML as a PDF using Chromium on Gotenberg container
+    def export
+      html = ActionController::Base.new.render_to_string render_config
+      gb = GotenbergClient.new
+      gb.render_to_pdf(html)
+    end
```

Comment:
Note: Future PR will cache this response based on the last update date of the form